tests/beaker_tests/vni/test_vni.rb

```diff
@@ -0,0 +1,197 @@
+###############################################################################
+# Copyright (c) 2015 Cisco and/or its affiliates.
```

Comment:
We are now using a new convention for our top_level directory names.  Since this is a cisco provider rename the top-level `vni` directory to be `cisco_vni`

` tests/beaker_tests/cisco_vni/test_vni.rb`